README.md

```diff
@@ -35,6 +35,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
   * [Game Development](#game-development)
   * [GUI library](#gui-library)
   * [HTML/XML parsing](#htmlxml-parsing)
+  * [HTML Builders](#html-builders)
```

Comment:
Please move it above "HTML/XML parsing" (we force alpha order here)

Reply:
Ok. Done.